app/helpers/layout_helper.rb

```diff
@@ -347,6 +347,19 @@ def fullscreen_button(element = "$(this).prev()")
   private
 
   def authorized_associations(associations)
-    associations.included_modules.include?(Authorizable) ? associations.authorized : associations
+    if associations.included_modules.include?(Authorizable)
+      if associations.respond_to?(:klass)
+        associations.authorized(authorized_associations_permission_name(associations.klass), associations.klass)
+      else
+        associations.authorized(authorized_associations_permission_name(associations), associations)
+      end
+    else
+      associations
+    end
+  end
+
+  def authorized_associations_permission_name(klass)
+    # TODO: Substitute by find_by :name for Rails 4
```

Comment:
This isn't true, find_by_* is not deprecated.  It keeps getting confused with find_all_by, which is: http://edgeguides.rubyonrails.org/4_0_release_notes.html#active-record-deprecations